_docs/data-model/v1/upload.md

```diff
@@ -55,6 +58,58 @@ At some point, we may support real-time monitoring of devices, in which case the
 
 The Tidepool platform supports the idea that a user other than the data owner may have upload permissions (for example, a clinic or a parent). The byUser field should be the actual userId of the user doing the upload. This field will be stored in encrypted form in the actual data stream as it constitutes PII.
 
+## device information
+
+For all of the fields under device information, we are trying to provide a basic taxonomy and standards so that it will be easy to query for specific device types or classes.
+
+Thus, if you are storing information in the Tidepool platform and are unsure of what information should go here, please contact us so we can help you and maintain a high standard of data quality.
+
+### deviceTags
+This is a list of broad attributes that can be applied to the device. It is an array of individual string values. Please use exactly the attributes below if they apply, in addition to any others that are appropriate:
+
+* `"insulin-pump"` (for insulin pumps)
+* `"cgm"` (for continuous glucose monitors)
+* `"bgm"` (for blood glucose meters that are not continuous meters)
+* `"fgm"` (for 'flash' glucose meters that are not continuous meters but able to read on demand)
+* `"pen"` (for insulin injection devices that record the doses)
+* `"manual"` (for manually entered data)
+
+Integrated devices or multifunction devices should use multiple tags as appropriate. For devices that do not fit in these categories, please contact Tidepool so we can standardize on a good tag.
+
+### deviceBrand
+
+This is an array containing the name or names of the device manufacturers. We are trying to standardize this field as well, so please use exactly the following names:
+
+    Abbott
+    Asante
+    Dexcom
+    Insulet
+    Medtronic
+    LifeScan
+
+Again, for new devices please contact Tidepool so we can standardize on a name.
+
+### deviceModel
+
+This is the name the manufacturer uses to identify the device. Examples might be:
+
+    MiniMed 530G
+    G4
+    FreeStyle Precision Xtra
+
+Again, this is an array so that devices that are created in partnership between multiple manufacturers can be properly specified.
```

Comment:
Hmm, I didn't see this one as an array. The Animas Vibe does not also need to include the `G4 Platinum` designation; in fact, I think that's probably inaccurate, because `G4 Platinum` implies the transmitter + receiver combo, but the Animas Vibe only includes the former, and in particular does not include the same algorithm in the receiver as new Dexcom G4 systems do.